packages/shared/components/inputs/RecipientInput.svelte

```diff
@@ -4,8 +4,8 @@
     import { RecipientAccountSelector, TextInput, InputContainer, Modal } from 'shared/components'
     import { activeProfile } from '@core/profile'
     import { getNetwork, NetworkType, nodeInfo } from '@core/network'
-    import { IAccountState } from '@core/account'
 
+    import { IAccountState } from '@core/account'
     export let recipient: string | IAccountState
```

Comment:
We should leave a line space here for better readability.

Reply:
I swear I just fixed these